src/views/htmlcontent/src/css/color-theme-dark.css

```diff
@@ -28,6 +28,10 @@ slick-grid.active .grid .slick-cell.selected {
   background-color: var(--color-cell-bg-grid-selected);
 }
 
+.slick-cell .grid-cell-value-container.missing-value {
+  color: var(--color-content) !important;
```

Comment:
To help me understan how this works:
- I see this color used for the .header css? Is that css used for all cells?
- What does `!important` do?